bob-apps/bin/bobd.rs

```diff
@@ -102,7 +105,16 @@ fn spawn_signal_handler(
 
 fn get_matches<'a>() -> ArgMatches<'a> {
     App::new(env!("CARGO_PKG_NAME"))
-        .version(env!("CARGO_PKG_VERSION"))
+        .version(
```

Comment:
I think it's better to use other arg/subcommand instead of version

Reply:
Why? It's supposed to be the main version of the app...

Reply:
The idea was to print all this info by executing `version` command